proof/refine/ARM/Ipc_R.thy

```diff
@@ -2869,99 +2869,398 @@ crunches cancelIPC
   for ResumeCurrentThread_imp_notct[wp]: "\<lambda>s. ksSchedulerAction s = ResumeCurrentThread
                                           \<longrightarrow> ksCurThread s \<noteq> t"
 
+lemma tcbEPFindIndex_inv[wp]:
```

Comment:
I think I moved a few of these lemmas in my PR for `receiveSignal_corres`. So you might need to hunt around for some of these after you've rebased. Soz